Graphite/GraphiteClient.cs

```diff
@@ -153,9 +159,17 @@ public async Task SendAsync(ICollection<Datapoint> datapoints, CancellationToken
 
         private async Task SendInternalAsync(ICollection<Datapoint> datapoints, CancellationToken cancellationToken)
         {
-            using (var client = new TcpClient(AddressFamily.InterNetworkV6))
+            TcpClient client;
+            if (UseDualStack)
+            {
+                client = new TcpClient(AddressFamily.InterNetworkV6) {Client = {DualMode = true}};
+            }
+            else
+            {
+                client = new TcpClient();

```

Comment:
Are you positive that the Azure issue does not require explicit use of `AddressFamily.Internetwork`?

Reply:
Yes, I am